### DISCUSSION TITLE
How to install custom plugin (OSF file)?

### DISCUSSION THREAD

**Discussion by Madeo19** [osmandapp/OsmAnd#19564] [Q&A] [answered]
Hi there,
just a simple question: how to install custom plugin (OSF file)? The docs do not tell anything on this. I cannot see any button for adding my plugin in the plugins menu (see screenshot). I use OsmAnd+ 4.6.13 on Android 8.0.0. I also tried to put the OSF file to the `Android/data/net.osmand.plus/files` folder.

<img src="https://github.com/osmandapp/OsmAnd/assets/50371436/bab81712-4e87-40eb-b38d-fb6895ef02f6" width="200"/>

Thanks for help!

**Comment by DmitryAlexei** [comment, marked as answer]
Please simply tap on your file and open it with OsmAnd

**Reply by Madeo19** [reply]
Works! Could we add this to the doc? I would contribute to, however, do not know the best way to do.
